api/app/src/command/medical/patient/get-patient.ts

```diff
@@ -19,6 +21,39 @@ export const getPatients = async ({
   return patients;
 };
 
+export const getPatientByDemo = async ({
+  facilityId,
+  cxId,
+  demo,
+}: {
+  facilityId: string;
+  cxId: string;
+  demo: PatientData;
+}): Promise<Patient | null> => {
+  const patient = await PatientModel.findOne({
+    where: {
+      cxId,
+      facilityIds: [facilityId],
```

Comment:
Shouldn't this be "contains", since the patient might be associated with more than one Facility?

Reply:
I was mimicking what we have for the `getPatients` but i can change it for both